package.json

```diff
@@ -18,8 +18,10 @@
     "watch:styles": "watch 'npm run style' ./src/app/styles",
     "publish": "node scripts/server.js",
     "test": "npm run test:unit && npm run test:integration",
-    "test:integration": "cd test && rspec && cd ..",
     "test:unit": "cd test/unit && NODE_PATH=$NODE_PATH:. BABEL_ENV=production mocha setup.js *.spec.js",
+    "test:integration": "cd test && rspec spec/integration_spec.rb && cd ..",
+    "test:proxy": "cd test && ruby spec/sauce/sauce_connect.rb && cd ..",
+    "test:browsers": "cd test && rspec spec/browser_spec.rb && cd ..",
```

Comment:
I kinda think a single `rspec` for all browsers is probably worse than looping through browsers somehow and calling `rspec` once on each...